asko_import/serializers.py

```diff
@@ -124,6 +124,13 @@ class Meta:
             "street_address",
         )
 
+    def __init__(self, *args, **kwargs):
+        exclude_fields = kwargs.pop("exclude_fields", None)
+        super().__init__(*args, **kwargs)
+        if exclude_fields:
+            for field_name in exclude_fields:
+                self.fields.pop(field_name, None)
+
```

Comment:
I feel like this change might have been left here accidentally? It's in the asko_import serializer.